lib/web/ui/labels.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ui
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/gravitational/teleport/api/types"
+)
+
+func makeLabels(labelMaps ...map[string]string) []Label {
```

Comment:
Can you add godoc to this method, `makeLabels` is a very generic name.

Reply:
I have added a comment with documentation!